portfolio/src/main/webapp/index.html

```diff
@@ -8,8 +8,14 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Grace's Website</h1>
+      <p> My name is Grace. I'm a STEP intern. Welcome to my website!</p>
+        <img src="images/propic.JPG" width = "300">
+      <p> Learn more about my random <a href="moments.html">moments</a> 
```

Comment:
nit: multiline html should put its content on the next line and properly indented:
```
<p>
  Some multi-line
  html content.
</p>
```